### DISCUSSION TITLE
a11y addon

### DISCUSSION THREAD

**Discussion by jjpxiii** [codegouvfr/react-dsfr#66] [General]
Bonjour / Hi (quelle langue ?! 😁)

Juste une précision, pour tester l'accessibilité des composants directement dans le storybook il y a un addon qui existe, que nous avons testé dans notre storybook et qui est très bien : https://storybook.js.org/addons/@storybook/addon-a11y

Il suffit de l'installer et de l'ajouter dans .storybook/main.js et l'onglet apparaîtra dans le panel du canvas.

Si c'est ok je peux créer une PR !

**Comment by garronej** [comment]
Yes please :))
Your PR would be very welcome!

**Reply by jjpxiii** [reply]
It can be used now in production, and I must say I'm impressed by the quality of the work that was put in this DS, not only the design is very neat and clean, but also the accessibility requirements are met for all the components I checked.
👏🏻👏🏻👏🏻👏🏻
